src/dynamodb/remove.js

```diff
@@ -0,0 +1,24 @@
+const { DeleteCommand } = require('@aws-sdk/lib-dynamodb');
+
+function ensureValidParameters(documentClient, tableName, key) {
+  if (!documentClient) throw new Error('documentClient is required.');
+  if (!tableName) throw new Error('tableName is required.');
+  if (!key) throw new Error('key is required.');
+  if (typeof key !== 'object') throw new Error('key should be an object.');
+}
+
+async function remove(documentClient, tableName, key, options = undefined) {
```

Comment:
Would it be confusing if the single one is called remove but the multi one is called batch-delete?

Reply:
I want to call it delete and not remove but js won't let me ;-) Do you think we should change to batchRemove instead of batchDelete?

Reply:
That's the question I wanted to raise, if we should call it batchRemove. Personally, not sure.. but it is weird to call them differently though :/ What do you think?

Reply:
Yeah we need to make a class to be able to use delete. So I think we should call them remove and batchRemove... It's at least consistent. Alternative is a class approach but that ship might have sailed 🤷‍♂️ . I vote for all-in remove!

Reply:
Yeah, I'll rename it to batchRemove